vote/src/vote_account.rs

```diff
@@ -55,6 +55,18 @@ pub struct VoteAccounts {
     >,
 }
 
+impl Clone for VoteAccounts {
+    fn clone(&self) -> Self {
+        Self {
+            vote_accounts: Arc::clone(&self.vote_accounts),
+            // Reset this so that if the previous bank did compute `staked_nodes`, the new bank
+            // won't copy-on-write and keep updating the map if the staked nodes on this bank are
+            // never accessed. See [`VoteAccounts::insert`] and [`VoteAccounts::staked_nodes`].
+            staked_nodes: OnceLock::new(),
```

Comment:
Without this change, once VoteAccounts::staked_nodes is initialized once - which it obviously is since we need to compute staked nodes at least once per epoch - then we always copy-on-write and update VoteAccounts::staked_nodes for every vote.